client/src/container/HomePage/index.js

```diff
@@ -1,2 +1,110 @@
-const HomePage = () => {};
+import React, { useState, useEffect } from 'react';
+import { notification, Button, Spin, Menu } from 'antd';
+import { MenuOutlined } from '@ant-design/icons';
+
+import { useHistory } from 'react-router-dom';
+// import { object } from 'prop-types';
```

Comment:
the component now is wrapping with react router so, we don't need it now.
delete these lines,